lib/src/main/java/macrobase/analysis/summary/itemset/result/EncodedItemsetResult.java

```diff
@@ -0,0 +1,37 @@
+package macrobase.analysis.summary.itemset.result;
+
+import java.util.Set;
+
+public class EncodedItemsetResult {
+    private double support;
+    private double numRecords;
+    private double ratioToInliers;
+    private Set<Integer> items;
+
+    public EncodedItemsetResult(double support,
```

Comment:
Though in this case, maybe we should rename EncodedItemsetResult to "ItemsetResult", no need for fpgrowth to know about "Encoding" at all, it just deals with ints. Then like peter suggested we can rename the current ItemsetResult to AttributeSetResult, no need for the summarizer to know that it is using itemset mining under the hood.